Neos.Flow/Classes/Cli/RequestBuilder.php

```diff
@@ -118,6 +118,13 @@ public function build($commandLine): Request
         $request = new Request();
         $request->setControllerObjectName(HelpCommandController::class);
 
+        if (is_array($commandLine) && in_array('--help', $commandLine, true)) {
+            $request->setControllerCommandName('help');
+            $request->setArguments(['commandIdentifier' => $commandLine[0]]);
+            $request->setControllerObjectName(HelpCommandController::class);
+            return $request;
+        }
```

Comment:
i would put this logic at below line 155 of `$commandIdentifier` so we can use this variable and also use `$rawCommandLineArguments` meaning we support the `$commandLine` being a string possibly.

Also the case `flow --help` will with this implementation result in an error as there is no command id named `--help`. The general help should be shown i suppose.

In the end this will make `--help` an reserved argument and we have to agree on that (which is likely but still breaking?)

Also this raw lookup against `--help` seems a little hacky. Like flow does support this short notation of boolean parameters but `--help=true` or `--help=false` are technically possible and will now still end up being passed to the final controller do we want that?

Reply:
> i would put this logic at below line 155 of $commandIdentifier so we can use this variable and also use $rawCommandLineArguments meaning we support the $commandLine being a string possibly.

So my check should also check if `$rawCommandLineArguments` is a string? Dont fully understand what you mean.

> Also the case flow --help will with this implementation result in an error as there is no command id named --help. The general help should be shown i suppose.

100%. Is there a way to trigger the default `flow help` via PHP?